2Peers-Backend/package.json

```diff
@@ -2,9 +2,9 @@
   "name": "2Peers-Backend",
   "version": "1.0.0",
   "description": "",
-  "main": "index.jsx",
```

Comment:
Can you change it back to jsx since we are removing the index.js file and app.js file

Reply:
Misunderstanding